### ISSUE TITLE
Hardwareunterstützung

### ISSUE THREAD

**Issue by Phantomias2006** [WLANThermo/WLANThermo_v3#6]
Sollen wir uns mit der v3 von Altlasten befreien?
Ich würde sogar soweit gehen nur mehr mini1 und mini2 zu unterstützen.
Also die Shutdowngeschichte der v3 weg und den ganzen v1 kram etc.
Das 4x20 Display und die Webcam-Funktion würde ich auch entfernen (zumindest vorläufig... Wir sehen/lesen ja wieviele diese funktion dann wirklich vermissen...)

**Comment by arminth** [comment]
Also das 4x20 Display würde ich nicht sterben lassen. Es gibt noch einige (ich auch), die das in Betrieb haben. Wenn du es raus wirfst, geht beim nächsten Update das Display nicht mehr. Schlecht. Weil das Update wird ja angezeigt in der gui.

**Comment by Phantomias2006** [comment]
Armin, die bist im Version 3 Git :-D
v3 möchte ich nicht mittels Update anzeigen. Das wird meiner Meinung nach eine neuinstallation. v2 läuft ja nachwievor auf den alten Thermos. Nur wird eben die Software nicht mehr gewartet ;-)

Ist aber nur mal meine Meinung dazu

**Comment by aschaefer85** [comment]
4x20 Display gehört ins Museum 😋

**Comment by Phantomias2006** [comment]
Ob und zu muss man(n) sich von altlasten Trennen... Pieppppp :-P

**Comment by kidbuster** [comment]
Öm... muss dann jeder seinen Thermo wieder aufmachen um an die MicroSD zu kommen für ne erneute Setup? Wenn ja, finde ich die Lösung mit dem Neuistallieren "suboptimal"
wäre nicht evtl ein Upgrade über das GUI oder SSH da nicht evtl doch Nutzerfreundlicher?

Angezeigt wird die Option dann nur, wenn man im jetzigen GUI die entsprechende Hardware ausgewählt hat.

**Comment by Phantomias2006** [comment]
Sicher nicht optimal aber dafür funktionsfähig :-)
Wir wollen hier gleich von Jessie auf Stretch wechseln und dieses Update kann gut gehen aber muss nicht. 
Ausserdem trennen wir uns damit auch von allen altlasten die im System noch vorhanden sind. Wie es wirklich wird können wir aber vermutlich erst abschätzen wenn die Software etwas weiter ist als jetzt :-)

**Comment by BjoernSch** [comment]
Selbst mit aufschrauben und Image schreiben wird alleine das Update auf Stretch schon um Größenordnungen schneller sein.

**Comment by kidbuster** [comment]
Ja, sind ja am ende eh fast alles nur Bastler, die das Thermo haben.
Wo steht ihr denn? GAAAAANZ am Anfang oder schon ein Stückchen weiter?^^ Ich weiß, coden ist ein haufen arbeit, ich bin nur neugierig, bekomm ich die Weihnachtsganz dieses Jahr etwa schon mit der neuen Firmware gegrillt? :D

**Comment by arminth** [comment]
Ich würde Kohlen nehmen!

:P

**Comment by BjoernSch** [comment]
Naja, Releasewechsel beim Betriebssystem macht man in der Regel per Neuinstallation weil sonst einfach zuviel Leichen übrigbleiben. Und der Releasewechsel ist gesetzt, so lange nach erscheinen des neuen wird das alte Release immer nicht mehr gepflegt, gerade bei Raspbian.
Auf die Weihnachtsgans würde ich nicht wetten.
